packages/manager/modules/pci/src/components/project/images-list/images-list.service.js

```diff
@@ -26,6 +26,14 @@ export default class ImagesList {
     this.$http = $http;
     this.OvhApiCloudProjectImage = OvhApiCloudProjectImage;
     this.OvhApiCloudProjectSnapshot = OvhApiCloudProjectSnapshot;
+    this.REGEX_PREFIX_BAREMETAL = REGEX_PREFIX_BAREMETAL;
+  }
+
+  removeBaremetalPrefix(imageType) {
+    if (imageType.match(this.REGEX_PREFIX_BAREMETAL)) {
+      return imageType.replace(this.REGEX_PREFIX_BAREMETAL, '');
+    }
+    return imageType;
```

Comment:
`replace` function won't return any error if `imageType` have no prefix occurences
```suggestion
    return imageType.replace(this.REGEX_PREFIX_BAREMETAL, '');
```

Reply:
It's applied